pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/AggregationFunctionFactory.java

```diff
@@ -77,6 +88,14 @@ public static AggregationFunction getAggregationFunction(FunctionContext functio
             // PercentileRawTDigest
             String percentileString = remainingFunctionName.substring(10);
             return new PercentileRawTDigestAggregationFunction(firstArgument, parsePercentileToInt(percentileString));
+          } else if (remainingFunctionName.matches("KLL\\d+")) {
+            // PercentileKLL
+            String percentileString = remainingFunctionName.substring(3);
+            return new PercentileKLLAggregationFunction(firstArgument, parsePercentileToInt(percentileString));
+          } else if (remainingFunctionName.matches("RAWKLL\\d+")) {
+            // PercentileRawKLL
+            String percentileString = remainingFunctionName.substring(6);
```

Comment:
Add comment and/or example to explain why 6? Ditto to all constant numbers used below.

Reply:
Same as above. "RAWKLL" has six characters, so we skip ahead to capture the rest of the function name.